sample/src/SampleAppButtonLaunch.js

```diff
@@ -24,23 +24,39 @@
 //
 
 import * as React from 'react';
-import { AzureAD, LoginType, MsalAuthProviderFactory } from 'react-aad-msal';
+import { AzureAD, LoginType } from 'react-aad-msal';
 import { basicReduxStore } from './reduxStore';
+import GetTokenButton from './GetTokenButton';
+
+// Import the authentication provider factory which holds the default settings
+import { authProviderFactory } from './authProviderFactory';
 
 class SampleAppButtonLaunch extends React.Component {
+  constructor(props) {
+    super(props);
+
+    // Change the login type to execute in a Popup
+    const provider = authProviderFactory.getAuthProvider();
+    provider.setLoginType(LoginType.Popup);
+  }
+
   unauthenticatedFunction = loginFunction => {
+    console.log('UNAUTHENTICATED');
```

Comment:
I noticed the other sample had console log statements so you could see what was happening. Not sure if that was a mistake or not, but I figured it was better to be consistent -- and these are actually pretty useful.